src/screens/OrgList/OrgList.test.tsx

```diff
@@ -233,11 +268,23 @@ describe('Organisation List Page', () => {
     );
     await wait();
 
-    //Search orgnizations with there name
-    const searchBar = screen.getByRole('textbox');
-    userEvent.type(searchBar, 'Akatsuki');
-    await wait();
+    // Test that the search bar filters organizations by name
+    const searchBar = screen.getByTestId(/searchByName/i);
+    const search1 = 'Akatsuki';
+    expect(searchBar).toBeInTheDocument();
+    userEvent.type(screen.getByTestId(/searchByName/i), search1);
+
+    // Test that the search bar is case-insensitive
+    userEvent.clear(searchBar);
+    const search2 = 'aKaTsUkI';
+    expect(searchBar).toBeInTheDocument();
+    userEvent.type(screen.getByTestId(/searchByName/i), search2);
+
+    // Test that the search bar filters all organization if there are is no search passed
+    userEvent.clear(searchBar);
+    const search3 = '';
```

Comment:
@aviraldevv 

Why have you used ```expect(searchBar).toBeInTheDocument();``` three times. 

The actual expect statements should check for the results after entering a query, and not the search bar itself.

Check the previous tests which were removed. They might give you an insight.

Reply:
The following expect statements need to be added

```expect(container.textContent).toMatch(search1);```
.
.
etc.

Reply:
Thanks for you input buddy,
I researched on this and found out that container method at times cannot capture the text content of the filtered search results due to the way they are being rendered in the DOM. In that case, using the text content property of a specific element that contains the search results may be a more reliable approach which I have used.